src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-transform: uppercase;
+  list-style: none;
+}
+
+.nav__item:nth-child(1) {
+  margin-left: 0;
+}
+
+.nav__link {
+  align-items: center;
+  line-height: 60px;
+  display: block;
+
+  font-weight: 500;
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  position: relative;
+
+  transition: all linear $main-during;
+
+  &:hover {
+    color: $main-color;
+  }
+}
+
+.is-active {
+  color: $main-color;
+}
+
+.is-active::after {
+  content: '';
+  height: 4px;
+  border-radius: 8px;
+  display: block;
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+  background-color: $main-color;
```

Comment:
Ensure that the SCSS variable `$main-color` is defined. This variable is crucial for the background color styling of the active link's underline.